### DISCUSSION TITLE
Word Frequency Feature & Frequency JSON format

### DISCUSSION THREAD

**Discussion by spirited1086** [FreeLanguageTools/vocabsieve#61] [Q&A] [answered]
I'm trying to test the word frequency feature in Vocabsieve. As a test I created a JSON file as follows:

```
{
  "frequency": ["el", "perro"]
}

```

I then importes this as a 'Frequency List.'

I tried it on a combination of three words, "el", "perro", and "amor," to see what would show up. But I see no difference or any additional information after enabling the frequency list.

Where is this information supposed to show up? And is there something wrong with my tested JSON format for the frequency list. The idea is once I get it working with two words, I'll then go about learning how to convert many other formats into the appropriate one for the frequency list.

**Comment by 1over137** [comment, marked as answer]
JSON is allowed to contain an array instead of an object. The file content should just be:
```json
["el","perro"]
```
The information can be displayed in two ways, either in terms of builtin stars (more stars meaning more frequent) or rank (displaying a number on the field). Realistically you want to use an existing list, converting the format if necessary. Frequency lists are allowed to be lemmatized (containing only dictionary forms) or not (containing all forms). Be sure to set the option for this correctly with respect to your list.
